src/pages/components/toggle.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { graphql } from 'gatsby';
+import { NamespacesConsumer } from 'react-i18next';
+import { withI18next } from '@wapps/gatsby-plugin-i18next';
+
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import { Toggle } from 'components/ExampleComponents';
+import ComponentDescription from 'components/ComponentDescription';
+import sideNavData from 'config/sidenav/components';
+import NoteBox from 'components/NoteBox';
+import Section from 'components/Section';
+import ComponentExample from 'components/ComponentExample';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+
+const Page: React.FC = (): React.ReactElement => {
+  const [isChecked, setChecked] = useState(false);
+
+  const getAriaLabelText = (t: Function): string =>
```

Comment:
This now gets the aria label for Toggle component => naming? It's also possible to use the translationHook for getting t for the function: https://react.i18next.com/latest/usetranslation-hook. This way the get aria label could work without needing the translate function as a parameter. Not sure if it would be better / worth the change though.